integrations/test_lightning.py

```diff
@@ -165,7 +165,6 @@ def _assert_called(model, stage):
         limit_val_batches=2,
         limit_test_batches=2,
         max_epochs=1,
-        progress_bar_refresh_rate=0,
```

Comment:
you can set `enable_progress_bar=False` as a 1:1 replacement for this line

Reply:
yes, but this is not really needed for this example :)